.github/workflows/CD_release_npm.yml

```diff
@@ -31,10 +31,13 @@ jobs:
 
       - run: |
           yarn install
-          yarn build
+          yarn build-mjml-react
 
       - name: Run semantic release (release + bump version)
-        run: npx semantic-release
+        run: |
+          cd dist
```

Comment:
As far as I could tell semantic-release doesn't allow specifying a path to package.json, so instead we run it from the same folder as the package.json in dist